### PR TITLE
backport zenodo release CI

### PR DESCRIPTION
**Pull request by johrstrom**
backport zenodo release CI so that I don't have to manually upload in case we continue to fix bugs in 2.0.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1202302957366374) by [Unito](https://www.unito.io)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        id: version
+        run: echo ::set-output name=version::${GITHUB_REF#refs/*/v}
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Ruby using Bundler
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "2.7.1"
```
Comment (Oglopf) [inline .github/workflows/release.yml:23]: I'm probably confused, but I thought we used ruby 2.7.2?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        env:
+          VERSION: ${{ steps.version.outputs.version }}
+      - name: Upload to Zenodo
+        uses: rseng/zenodo-release@0.0.17
+        with:
+          token: ${{ secrets.OSC_ROBOT_ZENODO_TOKEN }}
+          version: ${{ steps.version.outputs.version }}
+          zenodo_json: .zenodo.json
+          archive: packaging/v${{ steps.version.outputs.version }}.tar.gz
+          doi: '10.5281/zenodo.6323791'
```
Comment (gbsoftwaresolutions) [inline .github/workflows/release.yml:37]: lint stuff here.

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      "name": "Trey Dockendorf"
+    },
+    {
+      "affiliation": "Ohio SuperComputer Center",
+      "name": "Alan Chalker"
+    }
+  ],
+  "keywords": ["gateway", "hpc", "ruby", "rails"],
+  "license": "MIT"
+}
```
Comment (gbsoftwaresolutions) [inline .zenodo.json:29]: lint stuff here.

**Review by gbsoftwaresolutions — COMMENTED** [review]
looks good, just some lint stuff.

**Review by Oglopf — APPROVED** [review]
LGTM

File: .zenodo.json (hunk 3 above)
Comment (johrstrom) [reply .zenodo.json]: done

**Pull request merged**